biz.aQute.repository/src/aQute/maven/nexus/provider/Signer.java

```diff
@@ -71,8 +71,7 @@ public static int sign(File f, String cmdName, String passphrase, File output) t
 	public static int sign(File f, String cmdName, String key, String passphrase, File output) throws Exception {
 
 		Command cmd = new Command();
-
-		cmd.add(cmdName);
+		cmd.add(cmdName.split(" "));
```

Comment:
This seems dangerous since the command could have quoted elements. You  want:

```java
cmd.full(cmdName);
```

`full` will safely split the argument.

Reply:
Thanks for the advice! I used the constructor of Command, because it uses full internally.